module-cheatsheets/scRNA-seq-cheatsheet.md

```diff
@@ -113,6 +120,17 @@ Documentation for each of these packages can be accessed by clicking the package
 <br>
 <br>
 
```

Comment:
There was a bad page break here, so let's make one explicit maybe?  There may be other places we want to do this, but I didn't look fully through all the outputs yet, given some of the other suggested changes.
```suggestion
<div style="page-break-after: always;"></div>
```